baselines/run.py

```diff
@@ -91,7 +91,7 @@ def build_env(args):
                                    inter_op_parallelism_threads=1))
 
         if args.num_env:
-            env = SubprocVecEnv([lambda: make_mujoco_env(env_id, seed + i if seed is not None else None, args.reward_scale) for i in range(args.num_env)])    
+            env = SubprocVecEnv([(lambda y: (lambda: make_mujoco_env(env_id, seed, y, args.reward_scale)))(i) for i in range(args.num_env)]) 
```

Comment:
why double lambda and not:
`[lambda: make_mujoco_env(env_id, seed, i, args.reward_scale) for i in range(args.num_env)] ` ?

Reply:
ah, got it, because of this: https://stackoverflow.com/questions/6076270/python-lambda-function-in-list-comprehensions. ok. Still though, see the comment to cmd_util, maybe this can be addressed differently.